fs/operations/operations.go

```diff
@@ -2091,6 +2091,21 @@ func SetTier(ctx context.Context, fsrc fs.Fs, tier string) error {
 	})
 }
 
+// SetTierFile changes tier of a single file in remote
+func SetTierFile(ctx context.Context, o fs.Object, tier string) error {
+	objImpl, ok := o.(fs.SetTierer)
+	if !ok {
+		fs.Errorf(o, "Remote object does not implement SetTier")
+		return errors.New("remote object does not implement SetTier")
+	}
+	err := objImpl.SetTier(tier)
+	if err != nil {
+		fs.Errorf(o, "Failed to do SetTier, %v", err)
+		return err
```

Comment:
You don't need to log an Error and report it. Do one or the other :-)

Reply:
Oops, got it!